web/src/lib/async-content-fetchers/fetchTaskByFilename.ts

```diff
@@ -51,7 +52,11 @@ const fetchTaskFile = async (filename: string): Promise<string> => {
       try {
         file = await import(`@businessnjgovnavigator/content/roadmaps/license-tasks/${filename}.md`);
       } catch {
-        file = await import(`@businessnjgovnavigator/content/roadmaps/municipal-tasks/${filename}.md`);
+        try {
+          file = await import(`@businessnjgovnavigator/content/roadmaps/municipal-tasks/${filename}.md`);
+        } catch {
+          file = await import(`@businessnjgovnavigator/content/roadmaps/raffle-bingo-steps/${filename}.md`);
+        }
```

Comment:
[dust] This might actually be a case where `Promise` chaining via `.catch()` would be preferable to prevent nesting.